### PR TITLE
added tests to ensure everything is working fine

### PR DESCRIPTION
**Pull request by varaprasadh**
Test-Driven development is so important in software engineering, so I added the jest test library to the project,

Screenshot from 2021-08-18 19-56-50

now noob coders can contribute test cases as well.

### CODE REVIEW

File: __test__/index.test.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+*/
+
+const isEven = require("./index.tiny");
+
+describe('check if a number is even or odd', () => {
+    it("1 should be false", ()=>{
+        expect(isEven(1)).toBeFalsy();
+    }) 
+    it("2 should be true", ()=>{
+        expect(isEven(1)).toBeFalsy();
```
Comment (seeplusplus) [inline __test__/index.test.js:16]: this test doesn't even do what you say it does, are you even trying?

**Review by seeplusplus — CHANGES_REQUESTED** [review]

**Comment by darrylhodgins** [comment]
Branch coverage is a little low.

File: __test__/index.test.js (hunk 1 above)
Comment (varaprasadh) [reply __test__/index.test.js]: lol i was watching Samuel miller videos when i was doing this,just pushed a fix.

**Comment by varaprasadh** [comment]
> Branch coverage is a little low.

let the noob coders contribute test cases.

**Review by varaprasadh — COMMENTED** [review]
updated test case

**Review by thejoaov — APPROVED** [review]

**Review by seeplusplus — APPROVED** [review]

**Pull request closed without merging**